stdlib/public/stubs/DispatchShims.mm

```diff
@@ -104,6 +112,15 @@
 }
 
 void
+swift::_swift_dispatch_group_notify(
+	__swift_shims_dispatch_group_t group,
+	__swift_shims_dispatch_queue_t queue,
+	__swift_shims_dispatch_block_t block)
+{
+	dispatch_group_notify((dispatch_group_t)group, cast(queue), cast(block));
```

Comment:
I already tried it, and yeah, I got a C++ error because multiple types are actually just typedefs of `id`. There's a reason why we already have types that are omitted from the `cast` stuff.